apps/remix-ide/src/app/plugins/templates-selection/templates.ts

```diff
@@ -49,13 +51,15 @@ export const templates = (intl) => {
           value: "ozerc1155",
           displayName: "ERC1155",
           tagList: ["Solidity"],
+          description: "A standard interface for contracts that manage multiple token types",
           opts: {
             mintable: true
           }
         },
         {
           value: "ozerc20",
           displayName: "ERC20",
+          description: "A standard interface for fungible tokens",
```

Comment:
to distinguish it with other one, we can make it like: `A standard interface for mintable fungible tokens` and similarly

Reply:
we have tags there, no need to repeat it in description